sdk/core/core-amqp/src/connectionConfig/connectionConfig.ts

```diff
@@ -95,7 +95,7 @@ export namespace ConnectionConfig {
     const result: ConnectionConfig = {
       connectionString: connectionString,
       endpoint: parsedCS.Endpoint,
-      host: parsedCS && parsedCS.Endpoint ? (parsedCS.Endpoint.match("sb://([^/]*)") || [])[1] : "",
+      host: parsedCS && parsedCS.Endpoint ? (parsedCS.Endpoint.match(".*://([^/]*)") || [])[1] : "",
```

Comment:
Updated this at the places wherever `sb` is expected as the scheme except for the atomXML helpers.

Reply:
Ignore testhub for sure, it was meant for perf/stress testing purposes.
Let's stay away from the atom helpers for now as well